documentation/asset-tracker/topics/assettracker-hardware.md

```diff
@@ -0,0 +1,194 @@
+
+- Rack side panel
+- Rack door
+- Rack front panel
+
+In all these cases please:
+
+- Verify there is enough space for the modules and 3M adhesive.
+- Verify there is enough space to mount and remove equipment from the rack.
+- Verify the door can be opened and closed without interference.
+- Verify that there is not interference with power, network and other infrastructure.
+- Verify that the AssetTracker Strip is supported on a flat surface with no mechanical stress or pressure on it.
+
+Use the adhesive where possible and if needed use plastic zip ties for re-enforcement. Take care not to add pressure, puncture the casing or create mechanical stress on the assembled device.
+
+```{figure} /asset-tracker/media/installation_example_1.png
+:alt: Master Module and SATA cable
+:class: border-black
+
+Rack installation example
+```
+
+### Assembling Modules
+
+Identify the Master Module. The Master module has a SATA port above the top-most magnet position. The Extension Modules have male connectors on the top-end.
+
+```{figure} /asset-tracker/media/master_module_and_cable.png
+:alt: Master Module and cable
+:class: border-black
+
+Master Module and SATA Cable
+```
+
+Lay the modules on a flat surface and snap the male connector Extension Module to the female connector of the Master Module. Continue connecting Extension Modules ensuring that the Master Module is the top-most module in the chain.
+
+```{figure} /asset-tracker/media/master_and_extension.png
+:alt: Master Module and Extension
+:class: border-black
+
+Master Module and Extension Module ends
+```
+
+### Installing Modules
+
+Once the AssetTracker Strip is assembled and installed in the selected position, it needs to be connected to a Expansion Hub.Connect the RJ45 to SATA cable to the Master Module (SATA Port) and then to the Expansion Hub (V5 Port) with the RJ45 to SATA Cable. Secure the Expansion hub to the side using the provided 3M adhesive tape or another appropriate means.
+
+```{figure} /asset-tracker/media/master_module_to_hub_1.png
+:alt: Master Module to Expansion Hub example 1
+:class: border-black
+
+Master Module to Expansion Hub example 1
+```
+
+```{figure} /asset-tracker/media/master_module_to_hub_2.png
+:alt: Master Module to Expansion Hub example 2
+:class: border-black
+
+Master Module to Expansion Hub example 2
+```
+
+Connect the _RS485 In_ interface of the Expansion Hub to the _U Level_ port of the Gateway.
+
+```{figure} /asset-tracker/media/expansion_hub_to_gateway_1.png
+:alt: Expansion Hub to Gateway example 1
+:class: border-black
+
+Expansion Hub to Gateway example 1
+```
+
+```{figure} /asset-tracker/media/expansion_hub_to_gateway_2.png
+:alt: Expansion Hub to Gateway example 2
+:class: border-black
+
+Expansion Hub to Gateway example 2
+```
+
+If the Gateway a _V5008 AIoT_, Up to five total Hubs can be daisy chained together. Connect the _RS485 Out_ Port from the Hub connected to the Gateway to the _RS485 In_ Port of an additional Hub. Note the following:
```

Comment:
I think this sentence was meant to be:
"If the Gateway is a _V5008 AIoT_, up to five total Hubs can be daisy chained together."

Reply:
Fixed, thanks!